packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -265,13 +265,17 @@ class UserMessages {
       'for information about installing additional components.';
   String flutterNoMatchingDevice(String deviceId) => 'No supported devices found with name or id '
       "matching '$deviceId'.";
-  String get flutterNoDevicesFound => 'No devices found';
+  String get flutterNoDevicesFound => 'No devices found.';
```

Comment:
@christopherfujino how do we decide what strings to put in `user_messages`? I think it's primarily the few strings that need to be overridden by g3?
(see `google3_user_messages.dart` in cs).

Reply:
yes, that was the intent of the `UserMessages` class.

Reply:
I see, perhaps I should move not-overridden ones into the file they are used and make them consts?

Reply:
SGTM